concent_api/command_line_tool/concent_request.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import argparse
+import json
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from message_handler import MessageHandler
+from message_extractor import MessageExtractor
+from key_manager import KeyManager
+
+
+def get_json_data(message_file, message_str):
+    if message_file:
+        return json.load(open(message_file))
+    else:
+        return json.loads(message_str)
```

Comment:
There should be an assert here that `message_str is str`

Reply:
Error handling will be added in next iteration.